Makefile.PL

```diff
@@ -71,3 +79,10 @@ WriteMakefile(
         },
     },
 );
+
+if (!eval { ExtUtils::MakeMaker->VERSION('6.6303') }) {
```

Comment:
I don't understand what you mean.  This line is not requiring that version of EUMM -- it's providing code that will work if this version *isn't* installed on the user side.  `TEST_REQUIRES` was not added until EUMM 6.63_03; on versions before that, `BUILD_REQUIRES` is the best we had.